loopchain/launcher.py

```diff
@@ -230,12 +228,12 @@ def start_as_tool(args):
     print_epilogue()
 
 
-def start_as_peer(args, node_type=None):
+def start_as_peer(args):
     print_prologue()
 
     # apply default configure values
     port = args.port or conf.PORT_PEER
-    radio_station_target = f"{conf.IP_RADIOSTATION}:{conf.PORT_RADIOSTATION}"
+    radio_station_target = None
```

Comment:
if `radio_station_target` is None, no more needed.

Reply:
But the citizen nodes still needs radio_station_target.

Reply:
aha! you are right.